Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -242,7 +242,7 @@ - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController
 			default:
 				break;
 		}
-		
+		self.checkout = self.applePayAuthorizationDelegate.checkout;
```

Comment:
Then why is the checkout a public property of the authorization delegate? According to that logic, it shouldn't even be visible. Likewise some other properties, too, I would think. I believe the design exposed it as readonly because, implicitly, it was expected to change and be updated after completion.